notebooks/1_table_oriented.ipynb

```diff
@@ -0,0 +1,620 @@
+{
```

Comment:
<p>I suppose the reason that you don't show that a column of a dataframe is a series, is because indexing is only explained in a later tutorial? </p><p>I would maybe consider to just show <code>df[col]</code> already here, which might give a more natural introduction to series = column of dataframe.</p><p> </p><p>And then in a second step we can still show that you can create one manually as well</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
<p>done</p>